packages/core/RNSentryCocoaTester/RNSentryCocoaTesterTests/RNSentryTests.mm

```diff
@@ -122,6 +122,36 @@ - (void)testCreateOptionsWithDictionaryAutoPerformanceTracingDisabled
     XCTAssertEqual(actualOptions.enableAutoPerformanceTracing, false, @"Did not disable Auto Performance Tracing");
 }
 
+- (void)testCreateOptionsWithDictionarySpotlightEnabled
```

Comment:
The same as in Android, let's add test with a custom url.

Reply:
Good catch @krystofwoldrich 👍 
I now realised that [the value may be a the url string](https://docs.sentry.io/platforms/react-native/migration/v5-to-v6/#enablespotlight-and-spotlightsidecarurl-replaced-by-spotlight). I've updated the cocoa code with cc5383f61b71bb09f7797f666e2c58e8e0009273 and will iterate with the rest of your feedback.